src/sentry/static/sentry/app/views/organizationEventsV2/relatedEvents.jsx

```diff
@@ -80,9 +82,9 @@ class RelatedEvents extends AsyncComponent {
                 eventSlug: `${item['project.name']}:${item.id}`,
               },
             };
-            const project = {slug: item['project.name']};
+            const project = projects.find(p => p.slug === item['project.name']);
```

Comment:
Is it possible project is not in `this.props.projects`?

Reply:
If they project isn't in their project list, I would hope we don't get events from it either.